classes/02class/exercises/c02-network01/afaryy/README.md

```diff
@@ -0,0 +1,48 @@
+# c02-network01
+
+## Commands Execution Output
+
+- Answers for `10.0.0.1/24`:
+
+  - IP in binary: `00001010.00000000.00000000 .00000001`
+  - Netmask in binary: `11111111.11111111.11111111.00000000` 
+  - Netmask in decimals: `255.255.255.0`
+  - private/public: `private`
+  - hosts: `254` (10.0.0.1-10.0.0.254)
+
+- Answers for `192.168.0.1/16`:
+
+  - IP in binary: `11000000.10101000 .00000000.00000001`
+  - Netmask in binary: `11111111.11111111 .00000000.00000000` 
+  - Netmask in decimals: `255.255.0.0`
+  - private/public: `private`
+  - hosts: `65534 ` (192.168.0.1/16-192.168.255.254)
+
+- Answers for `249.165.166.135/30`:
+
+  - IP in binary: `11111001.10100101.10100110.100001 11`
+  - Netmask in binary: `11111111.11111111.11111111.111111 00` 
+  - Netmask in decimals: `255.255.255.252`
+  - private/public: `public`
+  - hosts: `2` (249.165.166.133-249.165.166.134)
+
+- Answers for `236.68.223.18/32`:
+
+  - IP in binary: `11111001.10100101.10100110.100001 11`
```

Comment:
This one is incorrect (that was from the previous item).

Reply:
I am a little careless.I will correct it. Thanks.